var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -60,6 +72,7 @@ class Nwchem(Package):
     depends_on("blas")
     depends_on("lapack")
     depends_on("mpi")
+    depends_on("armcimpi", when="armci=armcimpi")
```

Comment:
I tried to build all variants of `armci` and some deps are missing:
```suggestion
    depends_on("armcimpi", when="armci=armcimpi")
    depends_on("libfabric", when="armci=ofi")
    depends_on("rdma-core", when="armci=openib")
```
`armci=ofi` will still fail to build until https://github.com/GlobalArrays/ga/pull/324 gets patched in though.